docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -0,0 +1,599 @@
+        type: 'relation',
+        relation: 'manyToOne',
+        target: 'author',
+      },
+    },
+  };
+
+  ```
+
+:::
+
+::: details Bidirectional use case example:
+
+  An article belongs to only one category but a category has many articles.
+
+  ```js
+  // .api/[api-name]/content-types/article/schema.json
+
+  const model = {
+    attributes: {
+      author: {
+        type: 'relation',
+        relation: 'manyToOne',
+        target: 'category',
+        inversedBy: 'article',
+      },
+    },
+  };
+
+
+  // .api/[api-name]/content-types/category/schema.json
+
+  const model = {
+    attributes: {
+      books: {
+        type: 'relation',
+        relation: 'oneToMany',
+        target: 'article',
+        mappedBy: 'category',
+      },
+    },
+  };
+  ```
+:::
+
+::::
+
+:::: tab Many-to-Many
+
+Many-to-Many relationships are useful when:
+
+- an entry from Content-Type A is linked to many entries of Content-Type B,
+- and an entry from Content-Type B is also linked to many entries from Content-Type A.
+
+Many-to-many relationships can be unidirectional or bidirectional. In unidirectional relationships, only one of the models can be queried with its linked item.
+
+::: details Unidirectional use case example:
+
+  ```js
+  const model = {
+    attributes: {
+      categories: {
+        type: 'relation',
+        relation: 'manyToMany',
+        target: 'category',
+      },
+    },
+  };
+  ```
+
+:::
+
+::: details Bidirectional use case example:
+
+An article can have many tags and a tag can be assigned to many articles.
+
+  ```js
+  // .api/[api-name]/content-types/article/schema.json
+
+  const model = {
+    attributes: {
+      tags: {
+        type: 'relation',
+        relation: 'manyToMany',
+        target: 'tag',
+        inversedBy: 'articles',
+      },
+    },
+  };
+
+
+  // .api/[api-name]/content-types/tag/schema.json
+
+  const model = {
+    attributes: {
+      articles: {
+        type: 'relation',
+        relation: 'manyToMany',
+        target: 'article',
+        mappedBy: 'tag',
+      },
+    },
+  };
+  ```
+
+:::
+
+<!-- ? not sure what to do with this note and the following example, that's why I commented them for now -->
+<!-- :::tip NOTE
+The `tableName` key defines the name of the join table. It has to be specified once. If it is not specified, Strapi will use a generated default one. It is useful to define the name of the join table when the name generated by Strapi is too long for the database you use.
+:::
+
+**Path —** `./api/category/models/Category.settings.json`.
+
+```js
+{
+  "attributes": {
+    "products": {
+      "collection": "product",
+      "via": "categories"
+    }
+  }
+}
+``` -->
+::::
+
+:::::
+
+#### Components
+
+Component fields let your create a relation between your Content Type and a Component structure. They accept the following additional parameters:
+
+<!-- ? what does the "that let you create a list of data" mean for the `repeatable` parameter means? -->
```

Comment:
maybe "a list of components". 
Lets say the component is `gps_coordinates` that is structured with 2 variables (lat and long).
Non repeatable use case: I have bike Content Type where I want to know the current position.
```
Bike : {
  currentPosition: gps_coordinates (non repeatable) -> it's an object
}
```

Repeatable use case: I have bike Content Type where I want to know everywhere is went.
```
Bike : {
  pastPositions: gps_coordinates (repeatable) -> it's an array of objects
}
```